egs/aishell/s10/local/run_chain.sh

```diff
@@ -67,7 +69,7 @@ if [[ $stage -le 3 ]]; then
 fi
 
 train_ivector_dir=
-if [[ $train_ivector ]]; then
+if [ "$train_ivector" = true ]; then
```

Comment:
If `train_ivector` is going to have only two values: `true` or `false`, then
you can use

```bash
if $train_ivector; then
fi
```